### PR TITLE
Fix Ws connection 

### PR DESCRIPTION
**Pull request by kroos47**

### CODE REVIEW

File: src/dripper/polkadot/PolkadotActions.ts [hunk 1]
```diff
@@ -134,16 +134,25 @@ export class PolkadotActions {
         try {
           logger.warn("⚠️First try failed, retrying with backup", e);
           if (this.backup_account) {
```
Comment (moraesjeremias) [inline src/dripper/polkadot/PolkadotActions.ts:136]: What about main account?

Comment (kroos47) [reply src/dripper/polkadot/PolkadotActions.ts:136]: main account is using existing instance, which won't get disconnected.

Comment (moraesjeremias) [reply src/dripper/polkadot/PolkadotActions.ts:136]: Still not getting it.
Does Polkadot Api creates a state on the API instance arround the sending (faucet) address?

If not, why not reuse the same client for both faucet addresses?

Comment (kroos47) [reply src/dripper/polkadot/PolkadotActions.ts:136]: Incase Abnormal closure or something happen for that api backup can be healthy.

**Review by moraesjeremias — APPROVED** [review]
LFG!! :fire:

**Pull request merged**